docs/gitlab.md

```diff
@@ -2,21 +2,21 @@
 
 Git Credential Manager supports [gitlab.com][gitlab] out the box.
 
-## Using on a another instance
+## Using on another instance
 
 To use on another instance, eg. `https://gitlab.example.com` requires setup and
 configuration:
 
 1. [Create an OAuth application][gitlab-oauth]. This can be at the user, group
 or instance level. Specify a name and use a redirect URI of `http://127.0.0.1/`.
-_Unselect_ the 'Confidential' option. Set the 'write_repository' and
-'read_repository' scopes.
+_Unselect_ the 'Confidential' option. Set the 'read_repository' and
+'write_repository' scopes.
 1. Copy the application ID and configure
-`git config --global credential.https://gitlab.example.com.GitLabDevClientId <APPLICATION_ID>`
+`git config --global credential.https://gitlab.example.com.gitLabDevClientId <APPLICATION_ID>`
 1. Copy the application secret and configure
-`git config --global credential.https://gitlab.example.com.GitLabDevClientSecret
+`git config --global credential.https://gitlab.example.com.gitLabDevClientSecret
 <APPLICATION_SECRET>`
-1. Configure authentication modes to include 'browser'
+1. Optional if you want to force OAuth:
```

Comment:
```suggestion
1. If you would like to use OAuth, configure authentication modes to include 'browser'
```

Reply:
@ldennington I don't think that wording quite works. In my experimentation if that option is provided only browser auth is authorized whereas if `browser` option is not provided user is prompted to select between browser, token, and username/password.

Reply:
Perhaps then we could say something like `Optional if you want to force browser auth`?

Reply:
Updated